rust/gui-client/src-frontend/components/ColorPalettePage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+const primaryColors = [
```

Comment:
It seems a bit strange to prefix all of these with `bg-` - are they only used for backgrounds?

Reply:
Or I suppose this is a dev component and not actually used anywhere?

Reply:
Yep, just a dev component to actually see all the colours.